mobile/src/main/java/butter/droid/fragments/ShowDetailFragment.java

```diff
@@ -88,7 +84,7 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         mRoot = inflater.inflate(R.layout.fragment_showdetail, container, false);
         ButterKnife.bind(this, mRoot);
-        if (VersionUtils.isJellyBean() && container != null) {
+        if (container != null) {
```

Comment:
here too, this was previously only called on jellybean and is now called all the time.